### PR TITLE
bruno: fix css issue with patch file

### PR DESCRIPTION
**Pull request by alienman5k**
## Description of changes
Fix a css issue in bruno due to electron version used in Nix as compared to bruno upstream that uses older EOL version.

## Things done
- Created a patch file with the fix
- Reference patch file to patches attribute list in the package definition

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by gepbird — CHANGES_REQUESTED** [review]
Thanks for you contribution, I tested on x86_64-linux, works as expected:

bruno-css

Just a nit with git: don't merge branches like master/nixos-unstable into a feature branch like this, instead rebase if necessary. There are no merge conflicts without that merge commit or rebasing, so it can be deleted.
You can delete the merge commit by resetting to the previous commit: `git reset --hard d6c9297e46493e307005ff313c63528994c32cdf` then push your local branch to github with `git push --force`.

File: pkgs/by-name/br/bruno/package.nix [hunk 1]
```diff
@@ -73,6 +73,10 @@ buildNpmPackage' rec {
     })
   ];
 
+  patches = [
+    ./scroll-width-fix.patch
+  ];
```
[the remark this answers is not in the document]
Comment (alienman5k) [reply pkgs/by-name/br/bruno/package.nix:78]: I'll make those changes tomorrow once I'm back.

**Review by gepbird — CHANGES_REQUESTED** [review]
Please make the aforementioned changes, you just pushed an empty commit (how?).

**Comment by alienman5k** [comment]
I have no idea how that happened. I was trying to clone and switch to my branch on a different machine but I was not able to. Only thing I recall was refreshing my fork.

**Comment by alienman5k** [comment]
It seems like refreshing the branch on my fork was not a good idea, it pulled all changes from master, while there are no conflicts not sure if that would be a good idea.
So I am thinking to start over and create a new branch to make the changes (and remember to not make any refresh of the branch with master), does that sound like a better plan? I just don't want to cause any issues with that might be some mistakes from my limited experience with git.

**Comment by gepbird** [comment]
> It seems like refreshing the branch on my fork was not a good idea, it pulled all changes from master, while there are no conflicts not sure if that would be a good idea.

What do you mean by "refreshing"? Just pulling your fix-bruno-tabs branch shouldn't create extra merge commits.

> So I am thinking to start over and create a new branch to make the changes (and remember to not make any refresh of the branch with master), does that sound like a better plan? I just don't want to cause any issues with that might be some mistakes from my limited experience with git.

Yes, creating a new branch, recreating the changes and opening a new PR can work. Make sure to create the branch from master (or nixos-unstable to avoid potential mass rebuilds).

I think `git checkout fix-bruno-tabs` and `git reset --hard d6c9297e46493e307005ff313c63528994c32cdf` would fix your issue, but as a novice git user I'd go with starting from scratch too.

**Comment by kashw2** [comment]
The following process should be applicable as far as all the git stuff goes:

When trying to bring in latest changes:
`git rebase upstream/master` This assumes that upstream is the remote name for the nixpkgs repo owned by the NixOS org. You can check this by running `git remote -v`. It should print out something similar to mine:

```
origin	git@github.com:kashw2/nixpkgs.git (fetch)
origin	git@github.com:kashw2/nixpkgs.git (push)
upstream	git@github.com:NixOS/nixpkgs.git (fetch)
upstream	git@github.com:NixOS/nixpkgs.git (push)
```

Following that you should be able to perform a force push to the current branch via `git push origin $(git branch --show-current) --force`.

This also assumes your local remote is called `origin`.

To fix the current merge commits in this PR. You can use an interactive rebase by running `git rebase -i HEAD~4`. This will provide you an interface with the commits in the branch.

You can either drop the commits by specifying `drop` instead of `pick` at the start of each commit, or you can squash them into the first commit (which would bring this PR up to date) making sure to remove the extra body of the commit when you do so it reads only: `bruno: fix css with patch file`

**Comment by alienman5k** [comment]
Tried rebase but it seems I messed up, so will start from new branch instead. But will do in a couple of hours

**Pull request closed without merging**